biothings/web/query/pipeline.py

```diff
@@ -100,52 +106,47 @@ async def _(*args, **kwargs):
         ) as exc:
             logger.debug("QueryPipelineInterrupt: %s", exc.data)
```

Comment:
We do log the exception here but no where else. Potential choice to add `logger.exception` for exceptions where we think that would be useful

Reply:
Based off your previous comment that's perfectly fine by me